core/templates/dev/head/forms/formBuilder.js

```diff
@@ -598,9 +603,22 @@ oppia.factory('rteHelperService', [
       // Returns a DOM node.
       createRteElement: function(componentDefn, customizationArgsDict) {
         var el = $('<img/>');
-        el.attr('src', componentDefn.iconDataUrl);
+        if (componentDefn.previewUrlTemplate) {
+          el.attr('src',
+            $interpolate(
+              componentDefn.previewUrlTemplate)(
+                angular.extend(customizationArgsDict,
+                  {
+                    explorationId: explorationContextService.getExplorationId()
```

Comment:
Yep. Note this is more a question of future-proofing things, though -- for example, it's conceivable that we'll need to use RTEs in the future for some of the config properties.